.github/workflows/release.yml

```diff
@@ -82,9 +84,10 @@ jobs:
         env:
           OAUTH_CLIENT_ID: ${{ secrets.oauth_client_id }}
           OAUTH_CLIENT_SECRET: ${{ secrets.oauth_client_secret }}
-      - run: pnpm make:linux --publish onTagOrDraft
+      - run: pnpm package:linux --publish onTagOrDraft
         env:
           GH_TOKEN: ${{ secrets.github_token }}
+      - run: pnpm prepare:remove-source-maps
```

Comment:
1. Why do we need to remove the source-maps?
2. Why is this one after the package and on others before the package?

Reply:
Good eyes re: 2 - that's an oversight on my part.  Updated

Reply:
re 1 - it was within the https://github.com/electron-react-boilerplate/electron-react-boilerplate example.  My assumption that they remove the source maps so that the packaged+minified code cannot be reconstructed.  Not a big deal for an open source project

Reply:
Exactly, and it may help with tracing back bugs reported by users, however I am not sure how we can help users provide these off the top of my head 🤔

Reply:
Yeah, not too sure.  I think our exception handling and electron-logger should be catching most user reported issues.

That said, would it be best for now to skip minimizing source code?